staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/controller.go

```diff
@@ -448,7 +448,8 @@ func CheckRequest(quotas []corev1.ResourceQuota, a admission.Attributes, evaluat
 
 		match, err := evaluator.Matches(&resourceQuota, inputObject)
 		if err != nil {
-			klog.Errorf("Error occurred while matching resource quota, %v, against input object. Err: %v", resourceQuota, err)
+			klog.ErrorS(err, "Error occurred while matching resource quota against input object",
+				"ResourceQuota", resourceQuota)
```

Comment:
```suggestion
				"resourceQuota", resourceQuota)
```
lowerCamelCase is suggested https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments